.github/workflows/actions-compile.yml

```diff
@@ -18,16 +18,32 @@ jobs:
       - name: add missing packages
         uses: ConorMacBride/install-package@v1
         with:
-          apt: libelf-dev libbpf-dev libbsd-dev libsysrepo-dev libcap-dev libmnl-dev libselinux-dev libyang-tools
+          apt: libelf-dev libbpf-dev libbsd-dev libcap-dev libmnl-dev libselinux-dev
       - name: recursive checkout (with iproute2 submodule)
         uses: actions/checkout@v4
         with:
           submodules: true
+      - name: libyang v2.1.148 from upstream
+        uses: actions/checkout@v4
+        with:
+            repository: 'CESNET/libyang.git'
+            ref: 'v2.1.148'
+      - name: make libyang v2.1.148 from upstream
+        run: mkdir build && cd build && cmake -DCMAKE_BUILD_TYPE:String="Release" .. && make -j $(nproc) && make install
+      - name: sysrepo v2.2.105 from upstream
+        uses: actions/checkout@v4
+        with:
+          repository: 'sysrepo/sysrepo.git'
+          ref: 'v2.2.105'
+      - name: make sysrepo v2.2.150 from upstream
+        run: mkdir build && cd build && cmake -DCMAKE_INSTALL_PREFIX:PATH=$(pwd)/bin -DCMAKE_BUILD_TYPE:String="Release" .. && make -j $(nproc) && make install
```

Comment:
what's about removing the path and doing a sudo make install ?

Reply:
i'm trying to run the action locally, once done i will try that,